pco/src/tests/recovery.rs

```diff
@@ -52,21 +52,11 @@ fn assert_recovers<T: NumberLike>(
   name: &str,
 ) -> PcoResult<()> {
   for delta_encoding_order in [0, 1, 7] {
-    for (fm, fq) in [
-      (
-        FloatMultSpec::Enabled,
-        FloatQuantSpec::Disabled,
-      ),
-      (
-        FloatMultSpec::Disabled,
-        FloatQuantSpec::Provided(14),
-      ),
-    ] {
```

Comment:
Why the behavior change on this unit test?

Reply:
At least the first spec here is no longer possible, and I think `Auto + Classic` cover this pretty well. But I might have reduced some coverage with the 2nd one, so maybe I/we should add a new recovery test to hit that case. Thanks for the post-hoc review.